book/src/pong_tutorial/pong_tutorial_02.md

```diff
@@ -0,0 +1,367 @@
+# Drawing the paddles
+
+Now let's do some drawing! But to draw something, we need something to draw. In 
+Amethyst, those "somethings" are called Entities, which are described by 
+Components.
+
+Amethyst uses Specs for its ECS (Entity-component system), which is a parallel 
+Entity-component system written in Rust. You can learn more about Specs in the 
+[The Specs Book][sb].
```

Comment:
I didn't work on the Specs book for a while, I hope it is good enough to link it here.